app/domain/authentication/authn_jwt/signing_key/fetch_public_keys_signing_key.rb

```diff
@@ -0,0 +1,24 @@
+
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # This class is responsible for parsing JWK set from public-keys configuration value
+      class FetchPublicKeysSigningKey
```

Comment:
Can this class be Command class?

Reply:
Strictly speaking this class is like other `Fetch*SigningKey` classes.
They are not command classes, have additional `fech_signing_key` method, and like this one in the nearest feature are created explicitly (not by injection) by factory.
So assuming this the class can be a command class but there is no reason for it.

Reply:
And it has `nil` dependency as other `Fetch*` classes...